tests/resources/service-templates/tosca-simple-1.0/node-cellar/node-cellar.yaml

```diff
@@ -309,7 +311,7 @@ policy_types:
       client connections cleanly and shut down services. 
     derived_from: aria.Workflow
     properties:
-      implementation:
+      function:
```

Comment:
i thought in the service-template even according to your view this should remain `implementation`, no?